docs/source/advanced/apollo-ast.mdx

```diff
@@ -42,24 +42,15 @@ val graphQLText = """
     }
 """.trimIndent()
 
-val parseResult = graphQLText.parseAsGQLDocument()
+val parseResult = Buffer().writeUtf8(graphQLText).parseAsGQLDocument()
```

Comment:
Even though I kind of like that everything uses a `BufferedSource` now, this is verbose. Anyone wanting the `String` overloads back, I'll add them.